### ISSUE TITLE
Bug: Shader Difference Megathread/Mega-issue

### ISSUE THREAD

**Issue by EddsNotDedd** [ChaosInitiative/Portal-2-Community-Edition#1112]
### Describe the bug

_"one megaissue would be great"_ - craftablescience

 The primary goal of this Issue is to document places where things look visually different in Campaign maps compared to Portal 2, which require fixing as the intent of P2CE is to 1:1 match the base game. 

I will begin with three examples and slowly yet diligently add them as I find them. Others are welcome to contribute as well.

See also my other issue #1109 for one specific one that I won't be mentioning here as it's partly the fault of an intentionally added feature.

### Issue Map

**sp_a2_laser_intro**: Incorrect Water
![a2_laser_intro2](https://user-images.githubusercontent.com/107334905/212411789-37c61676-7374-4277-bd12-c217a1581682.png)
![a2_laser_intro](https://user-images.githubusercontent.com/107334905/212411779-1bf60ff9-aaaf-4274-b249-a255b0c79d96.png)

**sp_a4_speed_tb_catch**: Weird gel pipe prop lighting
![a4_speed_tb_catch](https://user-images.githubusercontent.com/107334905/212411822-54766222-a583-4f86-a4f0-4b91563f7035.png)
![a4_speed_tb_catch2](https://user-images.githubusercontent.com/107334905/212411824-ecf62374-bcce-41a6-9f6c-32d9cea5e956.png)

Any map with gel, but as an example here's **sp_a3_speed_flings**: Odd looking paint
![a3_speed_vanilla](https://user-images.githubusercontent.com/107334905/212412782-c76f6289-26b5-479b-8d0f-2cbd02631b9e.png)
![a3_speed_flings](https://user-images.githubusercontent.com/107334905/212412788-4065fcbe-6248-4e06-b91d-6faef6b11134.png)

### To Reproduce

Load the maps where these shader alterations occur. There's no more steps than that.

**Comment by EddsNotDedd** [comment]
The gel pipe parts are apparently a side effect of intended behavior. A careful look into that will be needed.

**Comment by TeamSpen210** [comment]
Yep, in Portal 2's engine if you have bumpmaps on static props it disables per-vertex lighting, but CSGO allows both to work simultaneously. It could still be wrong though if the maps need to be recompiled to do the lighting properly...

**Comment by JJL772** [comment]
Water was fixed a little while ago.
